internal/api/front/perses/src/app/project/dashboard/dashboard-details/dashboard-details.component.ts

```diff
@@ -24,30 +29,75 @@ import { DashboardModel } from '../dashboard.model';
   templateUrl: './dashboard-details.component.html',
   styleUrls: ['./dashboard-details.component.scss']
 })
-export class DashboardDetailsComponent implements OnInit {
+export class DashboardDetailsComponent {
+
+  name: string = "";
   isLoading = false;
-  currentProject = '';
   dashboard: DashboardModel = {} as DashboardModel;
+  ngxData = new Map<string, NgxLineChartModel[]>();
+  currentProject = '';
 
-  constructor(private dashboardService: DashboardService,
+  // TEMP static options data for graph display
+  view: [number, number] = [600, 400];
+  showXAxis = true;
+  showYAxis = true;
+  gradient = false;
+  showLegend = true;
+  showXAxisLabel = true;
+  xAxisLabel = 'Time';
+  showYAxisLabel = true;
+  yAxisLabel = 'Value';
+  timeline = true;
+  colorScheme = {
+    domain: ['#9370DB', '#87CEFA', '#FA8072', '#FF7F50', '#90EE90', '#9370DB']
+  };
+
+  constructor(private service: DashboardService,
+              private feedService: DashboardFeedService,
               private readonly toastService: ToastService,
-              private readonly projectService: ProjectService) {
+              private readonly projectService: ProjectService,
+              private route: ActivatedRoute) {
+  }
+
+  ngOnInit(): void {
+    this.isLoading = true;
+
+    this.route.params.subscribe(params => {
```

Comment:
this.name is used in the other subscription 
To be sure it is well define when used, you should use combineLatest of both route name and project context, I can provide you example and guide you on that

Reply:
Actually I changed it to this.dashboardName since. Does this solve the ambiguity you were talking about ?

Reply:
yeah I think it should do it